patches/minecraft/net/minecraft/client/renderer/ItemRenderer.java.patch

```diff
@@ -18,17 +18,16 @@
           p_229111_4_.func_227861_a_(-0.5D, -0.5D, -0.5D);
           if (!p_229111_8_.func_188618_c() && (p_229111_1_.func_77973_b() != Items.field_203184_eO || flag)) {
              boolean flag1;
-@@ -105,7 +105,8 @@
-             } else {
+@@ -106,6 +106,8 @@
                 flag1 = true;
              }
--
```

Comment:
Am I beating a dead horse with this?

Reply:
Sorry I'm not sure what you mean by this comment...